iree/compiler/Codegen/LLVMCPU/LLVMCPULowerExecutableTarget.cpp

```diff
@@ -92,6 +83,11 @@ static StringRef sanitizePipelineString(StringRef input) {
   return input;
 }
 
+static bool lowerToVectors(IREE::HAL::ExecutableVariantOp variantOp) {
```

Comment:
Could you refactor this a little bit to have a method in `Utils` and say `isVMVX`. Use that to check for vectorization here.